[2024-09-08T22:49:28.477+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-08T22:49:28.492+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: selenium_test_dag.run_selenium_test manual__2024-09-08T22:49:27.748289+00:00 [queued]>
[2024-09-08T22:49:28.500+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: selenium_test_dag.run_selenium_test manual__2024-09-08T22:49:27.748289+00:00 [queued]>
[2024-09-08T22:49:28.501+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-09-08T22:49:28.516+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): run_selenium_test> on 2024-09-08 22:49:27.748289+00:00
[2024-09-08T22:49:28.520+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=79) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-08T22:49:28.521+0000] {standard_task_runner.py:72} INFO - Started process 80 to run task
[2024-09-08T22:49:28.521+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'selenium_test_dag', 'run_selenium_test', 'manual__2024-09-08T22:49:27.748289+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/teste.py', '--cfg-path', '/tmp/tmp116v6lau']
[2024-09-08T22:49:28.523+0000] {standard_task_runner.py:105} INFO - Job 11: Subtask run_selenium_test
[2024-09-08T22:49:28.575+0000] {task_command.py:467} INFO - Running <TaskInstance: selenium_test_dag.run_selenium_test manual__2024-09-08T22:49:27.748289+00:00 [running]> on host 353f82669597
[2024-09-08T22:49:28.654+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='selenium_test_dag' AIRFLOW_CTX_TASK_ID='run_selenium_test' AIRFLOW_CTX_EXECUTION_DATE='2024-09-08T22:49:27.748289+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-08T22:49:27.748289+00:00'
[2024-09-08T22:49:28.655+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-08T22:49:32.908+0000] {logging_mixin.py:190} INFO - Google
[2024-09-08T22:49:33.040+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-08T22:49:33.048+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-08T22:49:33.049+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=selenium_test_dag, task_id=run_selenium_test, run_id=manual__2024-09-08T22:49:27.748289+00:00, execution_date=20240908T224927, start_date=20240908T224928, end_date=20240908T224933
[2024-09-08T22:49:33.070+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-08T22:49:33.086+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-08T22:49:33.091+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
